test/liquid_voting_web/absinthe/mutations/create_delegation/after_creating_related_votes_test.exs

```diff
@@ -0,0 +1,183 @@
+defmodule LiquidVotingWeb.Absinthe.Mutations.CreateDelegation.AfterCreatingRelatedVotesTest do
+  use LiquidVotingWeb.ConnCase
+  import LiquidVoting.Factory
+
+  alias LiquidVotingWeb.Schema.Schema
+
+  describe "create proposal-specific delegation" do
+    test "after creating related proposal vote by delegate" do
+      # Insert vote and get related participant and proposal_url.
+      vote = insert(:vote, yes: true)
+      participant = vote.participant
+      proposal_url = vote.proposal_url
+
+      # Insert participant to act as delegator.
+      delegator = insert(:participant, organization_id: vote.organization_id)
+
+      # Create proposal_url delegation to participant (as delegate) and get
+      # voting result for proposal_url.
+      query = """
+      mutation {
+        createDelegation(delegatorEmail: "#{delegator.email}", delegateEmail: "#{
+        participant.email
+      }", proposalUrl: "#{proposal_url}") {
+          proposalUrl
+          votingResult {
+            inFavor
+            against
+          }
+        }
+      }
+      """
+
+      {:ok, %{data: %{"createDelegation" => delegation}}} =
+        Absinthe.run(query, Schema, context: %{organization_id: vote.organization_id})
+
+      assert delegation["proposalUrl"] == proposal_url
+      assert delegation["votingResult"]["inFavor"] == 2
```

Comment:
Maybe add a comment here as well, explaining the assertion that the vote now carries a weight of 2 due to the delegation given to the voter?